plugins/woocommerce/tests/e2e-pw/global-setup.js

```diff
@@ -15,6 +16,22 @@ module.exports = async ( config ) => {
 	console.log( `State Dir: ${ stateDir }` );
 	console.log( `Base URL: ${ baseURL }` );
 
+	// Try calling the site reset plugin (if testing on an external environment)
+	if ( ! baseURL.includes( 'locahost' ) ) {
+		console.log( 'Resetting site...' );
+		try {
+			const response = await axios.get(
```

Comment:
Playwright's `request` is preferred all over the suite instead of `axios`. We should be consistent.